pkg/minikube/cruntime/docker.go

```diff
@@ -365,6 +365,25 @@ func dockerImagesPreloaded(runner command.Runner, images []string) bool {
 	return true
 }
 
+// ForceSystemdCgroupManager forces docker to use systemd as cgroup manager, as recommended in k8s docs:
+// https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker
+func (r *Docker) ForceSystemdCgroupManager() error {
+	daemonConfig := `{
+"exec-opts": ["native.cgroupdriver=systemd"],
```

Comment:
it would be nicer if this function _only_ changed the native.cgroupdriver setting ?

instead of deleting and overwriting the entire docker daemon.json configuration file

We had lots of issues like this with CRI-O before, when we replaced the crio.conf
The Docker config hasn't change in ages, but who knows. Maybe a new driver ?

Reply:
So it looks like the daemon.json file doesn't exist at all currently. I switched to including it in the base image, instead of modifying it while minikube is starting.

Let's see how the integration tests respond, but I locally tested on my Macbook, my Linux workstation, and in Cloud Shell. It seems to be working normally on all three.